config/v1/types_scheduling.go

```diff
@@ -93,6 +96,29 @@ var (
 	NoScoring SchedulerProfile = "NoScoring"
 )
 
+// ProfileCustomizations contains various parameters for modifying the default behavior of certain profiles
+type ProfileCustomizations struct {
+	// dynamicResourceAllocation allows to enable or disable dynamic resource allocation within the scheduler.
+	// Dynamic resource allocation is an API for requesting and sharing resources between pods and containers inside a pod.
+	// Third-party resource drivers are responsible for tracking and allocating resources.
+	// Different kinds of resources support arbitrary parameters for defining requirements and initialization.
+	// Valid values are Enabled, Disabled and omitted.
+	// When omitted, this means no opinion and the platform is left to choose a reasonable default,
+	// which is subject to change over time.
+	// The current default is Disabled.
+	DynamicResourceAllocation DRAEnablement `json:"dynamicResourceAllocation"`
```

Comment:
We've lost the `+optional` tag here, please restore it

Reply:
Done